src/controller.rs

```diff
@@ -44,27 +44,25 @@ pub fn start(
         lang_srv = match language_server_transport::start(&lang.command, &lang.args, &lang.envs) {
             Ok(ls) => ls,
             Err(err) => {
-                // If we think that the server command is not from the default config, then we
-                // send a prominent error to the editor, since it's likely configuration error.
-                let might_be_from_default_config =
-                    !lang.command.contains('/') && !lang.command.contains(' ');
-                if might_be_from_default_config {
-                    panic!("{}", err);
-                }
-                let command = format!(
-                    "lsp-show-error {}",
-                    editor_quote(&format!("failed to start language server: {}", err)),
-                );
-                if to_editor
-                    .send(EditorResponse {
-                        meta: initial_request.meta,
-                        command: Cow::from(command),
-                    })
-                    .is_err()
-                {
-                    error!("Failed to send command to editor");
+                error!("failed to start language server: {}", err);
+                // If the server command isn't from a hook (e.g. auto-hover),
+                // then send a prominent error to the editor.
```

Comment:
Very cool. We couldn't show an error before because we want to support a global `lsp-enable`.
Now that we can track hooks we can better capture the user's intent.